Foundation/Class/Storable.hs

```diff
@@ -124,3 +128,9 @@ instance StorableFixed Word32 where
 instance StorableFixed Word64 where
     size      = primSizeInBytes . toProxy
     alignment = primSizeInBytes . toProxy
+instance StorableFixed (Ptr a) where
+    size      = Size . Foreign.Storable.sizeOf    . toUndefined
```

Comment:
probably easier to use directly here `(undefined :: Proxy a -> a)` instead of defining `toUndefined` globally to the module